modules/registry-refactoring/src/test/java/org/apache/airavata/registry/core/repositories/appcatalog/ApplicationInterfaceRepositoryTest.java

```diff
@@ -0,0 +1,107 @@
+package org.apache.airavata.registry.core.repositories.appcatalog;
```

Comment:
License header

Reply:
@DImuthuUpe thank you for the review. I have added the license header.